libs/php-commons/ErrorNotify.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Leaf;
+namespace Leaf;
```

Comment:
I would put this in the app/Leaf folder and leave the namespace as App\Leaf. We should be moving away from html/libs/php-commons

Reply:
Moved to the new location, updated refrences.